probe_scraper/transform_probes.py

```diff
@@ -182,10 +182,14 @@ def transform(probe_data, node_data, break_by_channel):
             print "  from: " + str({"node": node_id, "version": readable_version})
             for probe_type, probes in probe_data[channel][node_id].iteritems():
                 # Group the probes by the release channel, if requested
+                if probe_type not in result_data:
+                    result_data[probe_type] = 1
+                else:
+                    result_data[probe_type] +=1
                 extract_node_data(node_id, channel, probe_type, probes, result_data,
-                                  readable_version, break_by_channel)
-
-    return result_data
+                                 readable_version, break_by_channel)
+        new_data[channel] = result_data
+    return new_data
```

Comment:
We should return both `result_data` and `probes_counts` here.

Reply:
ok.